app/javascript/mastodon/polyfills/intl.ts

```diff
@@ -0,0 +1,105 @@
+// import { shouldPolyfill as shouldPolyfillCanonicalLocales } from '@formatjs/intl-getcanonicallocales/should-polyfill';
```

Comment:
@renchap does this file come from somewhere? Kinda curious about all the commented code here?

Reply:
No, I wrote it, but as explain in the relevant commit (I think) I wanted to let those here in case we notice they need to be enabled because we have user reporting it does not work in a given (old) browser.

I spent some time writing this and I did not wanted to redo it if it was needed :)